requirements.txt

```diff
@@ -12,6 +12,7 @@ contextlib2==0.6.0
 cryptography==2.8
 decorator==4.4.2
 Django==2.2.11
+django-axes==5.3.1
```

Comment:
Since you are adding a new package, you should add the package & appropriate info to the alphabetical list at `docs/developing/requirements.rst`.

In addition, this build will fail, because you did not run `python setup.py egg_info`